src/Hl7.Fhir.ElementModel/MaskingNode.cs

```diff
@@ -65,10 +65,12 @@ public MaskingNode(ITypedElement source, MaskingNodeSettings settings = null)
                 ies.ExceptionHandler = (o, a) => ExceptionHandler.NotifyOrThrow(o, a);
         }
 
-        private MaskingNode(MaskingNode parent, ITypedElement source)
+        private MaskingNode(MaskingNode parent, ITypedElement source, bool? includeAll = null)
```

Comment:
why not make the new parameter mandatory?  That will ensure every call site needs to be explicit about this, which is fine (it's a private method anyway).